rebuild/config/_icons.scss

```diff
@@ -0,0 +1,152 @@
+.ap-icon {
+    $self: &; // Notes: https: //css-tricks.com/using-sass-control-scope-bem-naming/
+
+    //Icon sizes
+    $icon-sizes: ('tiny': 16px, 'small': 24px, 'medium': 32px, 'ui': 40px, 'large': 88px, 'maxi': 152px);
+
+    //Icons and their file locations
+    $icons: (
+        'hamburger--primary': (
+            'asset': 'rebuild/icons/hamburger--primary.svg',
+        ),
+        'hamburger-close--primary': (
+            'asset': 'rebuild/icons/hamburger-close--primary.svg',
```

Comment:
Do we need this as a separate icon to https://github.com/AgePartnership/oleg-brand-kit/pull/70/files#diff-c7233cb673a761afb7daf902d681f23261621b4c31e6e23ba620746a0ed040d0 or can we consolidate?